pilot/cmd/pilot-agent/status/server.go

```diff
@@ -95,6 +101,19 @@ type Server struct {
 	envoyStatsPort      int
 }
 
+func init() {
+	registry := prometheus.NewRegistry()
+	wrapped := prometheus.WrapRegistererWith(map[string]string{"agent": "istio"}, prometheus.Registerer(registry))
+
```

Comment:
```suggestion
	wrapped.MustRegister(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{}))
	wrapped.MustRegister(prometheus.NewGoCollector())
```
I think we can add these back with the above change